## Molecular Characterization Initiative
CCDI’s [Molecular Characterization Initiative](https://ccdi.cancer.gov/MCI) (MCI) underlying raw sequencing, clinical, and imaging data can be searched through the [CCDI Hub Explore](https://ccdi.cancer.gov/explore?dbgap_accession=phs002790). Controlled-access data is available via dbGaP authorization under accession number [phs002790](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002790).

### Clinical Data
The clinical data, including Cancer Type and Cancer Type Detailed, were provided by the Children’s Oncology Group. MGMT Methylation Status, Methylation Diagnosis Subclass, Percent Tumor, and Percent Necrosis were sourced from reports provided by the Nationwide Children’s Hospital’s Steve and Cindy Rasmussen Institute for Genomic Medicine (NCH-IGM).

### Mutation Data
The MCI mutation data was sourced from somatic VCFs generated by NCH-IGM CLIA-compliant enhanced paired tumor–normal exome sequencing. 

The following steps were used to process the VCF files: 
1.	The VCF file format is first converted to MAF format using the vcf2maf tool, which calculates end position of the variant and strand information as well as transforms data from columns in the VCF file to match the standard MAF format. 
2.	The variants in MAF format are then annotated by Ensembl’s VEP (Variant Effect Predictor) tool (which is wrapped in the vcf2maf tool) which selects a single “effect” per variant and provides variant level annotations from a variety of other databases and sources such as rsID from dbSNP, amino acid change, variant classification (e.g. Silent, Missense, Nonsense, In Frame Insertion/Deletion etc.), and others.
3.	The individual MAF files were then concatenated into a single MAF for loading into cBioPortal. The concatenated MAF was then filtered to retain only mutations that were marked as passing all quality filters in the VCF. 
4.	The file was then annotated with two additional columns (Reported Significance and Reported Significance System), which represent AMP/ASCO/CAP categories of clinical significance. These data were sourced from Somatic Disease/Germline Comparator Exome reports from NCH-IGM. To perform this annotation, records were matched against available data, such as chromosome start position and reference allele.
    1. **Note:** *Discrepancies between variant location and molecular change information (such as left-shifting vs. right-shifting base calling) across the variant reports and exome MAF data may result in some Reported Significance data being omitted from display in the CCDI cBioPortal instance for MCI mutation data.*
5.	Upon data loading, cBioPortal automatically filters out variants classified as Silent, Intron, IGR, 3’UTR, 5’UTR, 3’Flank, and 5’Flank, except for promoter mutations of the TERT gene. Additionally, some variants were annotated as "Targeted_Region" after VEP annotation. This is a non-standard mutation type used as a catch-all for variants with multiple VEP consequences that do not map to recognized mutation types. As such, they were also excluded from analysis.

### Copy Number Data
Copy number variation (CNV) data generated using the GATK CNV pipeline were provided by NCH-IGM. Data processing and subsequent display in cBioPortal includes:

##### Parsing Segment-Level Data
CNV output files containing segment-level log2 copy ratio values were parsed and reformatted to the required cBioPortal tabular structure. Each segment was mapped to genomic coordinates along with its corresponding log2 value.

##### Gene-Level Mapping
Segment data were intersected with Gencode v48 gene locations to assign gene-level copy ratio values. For genes overlapping multiple segments, log2 values were assigned based on dominant coverage (>50% overlap) to produce gene-level continuous CNV profiles for each sample.

##### Discrete CNV Calls (GISTIC-like)
To enable downstream discrete CNV analyses, gene-level log2 fold change values were binned with GISTIC-like discrete thresholds to provide putative gain and loss calls at the gene level.
*	Amplification: log2 > +2.0 (CN > 8.0)
*	Gain: log2 > 0.3 (CN > 2.3)
*	Deletion: log2 < –0.3 (CN < 1.62) 
*	Homozygous Deletion: log2 < –1.0 (CN < 1)
